bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/CatalogSearchCriterion.scala

```diff
@@ -16,7 +16,7 @@ import Scalaz._
  * Used to query catalogs and filter and categorize query results.
  * See OT-20
  */
-case class CatalogSearchCriterion(name: String, referenceBand: MagnitudeBand, magRange: MagnitudeRange, radiusLimits: RadiusConstraint, offset: Option[Offset] = None, posAngle: Option[Angle] = None) {
+case class CatalogSearchCriterion(name: String, referenceBands: List[MagnitudeBand], magRange: MagnitudeRange, radiusLimits: RadiusConstraint, offset: Option[Offset] = None, posAngle: Option[Angle] = None) {
```

Comment:
This may not be that clear but it is a side effect of the multiple band selection logic. What this actually represents is that the `magRange` will be applied to the first band available on `referenceBands`, typically `[r, R, UC]` or `H`